tests/test_xmltools.py

```diff
@@ -3,14 +3,10 @@
 from ispyb.xmltools import mx_data_reduction_to_ispyb, xml_file_to_dict
 
 
-def test_mx_data_reduction_xml_to_ispyb(testdb):
+def xml_to_ispyb(testdb, filename):
```

Comment:
I think what you're looking for here is a [pytest parametrized test](https://docs.pytest.org/en/6.2.x/parametrize.html):

```suggestion
import pytest

@pytest.mark.parametrize("filename", ["data/mx_data_reduction_pipeline_results.xml", "data/autoPROC-two-ap_scaling_containers.xml"])
def test_mx_data_reduction_xml_to_ispyb(testdb, filename):
```

(not tested the above, but should hopefully be broadly correct)